.editorconfig

```diff
@@ -20,13 +20,3 @@ space_redirects    = true
 # Some scripts will only run with bash
 [{mkfadumprd,mkdumprd,kdumpctl,kdump-lib.sh}]
 shell_variant = bash
-
-# Use dracut code style for *-module-setup.sh
```

Comment:
Hi @prudo1,

Do I understand it correct that with this change, the kdump dracut modules won't adopt dracut code style?

Reply:
Oh, sorry for the notice. I didn't notice the editorconfig files in sub-folders until now.